test/tests/fvkernels/mms/mass-mom-mat-advection-diffusion/test.py

```diff
@@ -0,0 +1,18 @@
+import mms
+import unittest
+from mooseutils import fuzzyEqual
+
+class TestMassMomdAvectionDiffusion(unittest.TestCase):
+    def test(self):
+        labels = ['l2_rho','l2_vel']
+        df1 = mms.run_spatial('input.i', 4, y_pp=labels, mpi=2)
+        fig = mms.ConvergencePlot(xlabel='Element Size ($h$)', ylabel='$L_2$ Error')
+        fig.plot(df1, label=labels, marker='o', markersize=8, num_fitted_points=3, slope_precision=2)
+        fig.save('mmm-advection-diffusion.png')
+
+        for _,value in fig.label_to_slope.items():
+            print(value)
```

Comment:
```suggestion
```

Reply:
I m not against prints in python scripts when they are helpful for us to run them locally btw, but they need to have a message